python/GangaLHCb/__init__.py

```diff
@@ -85,43 +85,21 @@ def _store_root_version():
 
 
 def _store_dirac_environment():
+    from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache
     diracversion = _guess_version('LHCBDIRAC')
-    import tempfile
-    import subprocess
-    import os
     platform = os.environ['CMTOPT']
     setup_script = 'SetupProject.sh'
```

Comment:
Ok, I can make that change as a commit in this PR if you'd like? It's only about a 3-line change.